docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -327,3 +308,294 @@ An error can occur if the [list of ciphers](#ciphers) does not match between the
 
 **Solution:**  
 Review the supported TLS versions and ciphers used in both the client and the server.
+
+## Full example of AT-TLS configuration
+
+Review a full working example of an AT-TLS configuration file on z/OS, specifically used for defining secure communication between different services in a mainframe environment. All port values are examples.
+The example is commented for convenience. 
+<details>
+
+<summary>Click here to display the full AT-TLS configuration file.</summary>
+
+```pagent
+# Main inbound rule, all API ML core services have it defined.
+TTLSRule ApimlServerRule
+{
+  LocalAddr All
+  RemoteAddr All
+  LocalPortRange 7554-7559
+  Jobname ZWE*
+  Direction Inbound
+  TTLSGroupActionRef ServerGroupAction
+  TTLSEnvironmentActionRef ApimlServerEnvironmentAction
+  TTLSConnectionActionRef ApimlServerConnectionAction
+}
+
+# Example southbound service inbound rule
+TTLSRule ApimlDCServerRule
+{
+  LocalAddr All
+  RemoteAddr All
+  LocalPortRange 40030-40040 # Example service ports
+  Jobname ZWEDC*
```

Comment:
Zowe defaults to prefix ZWE1
https://github.com/zowe/zowe-install-packaging/blob/fdcdb2618080cf87031c070aed7e90503699ab5f/example-zowe.yaml#L306

So the jobname example should be ZWE1DC* instead